com.ibm.wala.ide.tests/build.gradle

```diff
@@ -24,16 +24,19 @@ eclipseMavenCentral {
 
 dependencies {
 	testImplementation(
-			'org.osgi:org.osgi.core:6.0.0',
+			'org.eclipse.platform:org.eclipse.osgi:3.15.100',
 			project(':com.ibm.wala.core'),
 			project(':com.ibm.wala.ide'),
 			project(':com.ibm.wala.util'),
 	)
 	testImplementation('org.eclipse.platform:org.eclipse.ui.workbench') {
 		version { strictly '3.120.0' }
 	}
-	testImplementation('javax.annotation:javax.annotation-api') {
-		version { strictly '1.3.2' }
-	}
-
 }
+
+configurations.all {
+	resolutionStrategy.dependencySubstitution {
+		substitute module('org.eclipse.platform:org.eclipse.osgi.services') using module('org.eclipse.platform:org.eclipse.osgi:3.15.100')
+		substitute module('xml-apis:xml-apis-ext') using module('org.eclipse.birt.runtime:org.w3c.css.sac:1.3.1.v200903091627')
+	}
+}
```

Comment:
Could we also add a comment on this?  And maybe a trailing newline?

Reply:
Added an explanation using the `because` construct that is optionally allowed here.

Added a trailing newline.